commands/create.go

```diff
@@ -109,8 +109,18 @@ func create(command *Command, args *Args) {
 	project := github.NewProject(owner, newRepoName, host.Host)
 	gh := github.NewClient(project.Host)
 
-	if gh.IsRepositoryExist(project) {
-		ui.Errorln("Existing repository detected. Updating git remote")
+	repo, err := gh.Repository(project)
+	if err != nil {
+		utils.Check(err)
```

Comment:
I think this breaks current CI checks. Keep in mind that `err` will be present if the repo doesn't exist. This whole conditional can go away.

Reply:
Yupp, this took me a bit to figure out why all the tests were breaking. It did occur to me that we don't currently differentiate between a "the repo doesn't exist" error, and a _real_ error. In that situation a duplicate error wouldn't be created (assuming the github api handles the conflict properly), however it could be dealt with better in hub itself.

It's not really a concern here though, and is outside the scope of what I'm trying to accomplish with this PR